_episodes_rmd/07-control-flow.Rmd

```diff
@@ -340,14 +340,14 @@ output_vector2
 > > We can do that as follows:
 > >
 > > 1. Loop over each of the unique values of 'continent'
-> > 2. For each value of continent, create a temporary variable storing the life exepectancy for that subset,
+> > 2. For each value of continent, create a temporary variable storing the that subset
 > > 3. Return the calculated life expectancy to the user by printing the output:
 > >
 > > ```{r 07-chall-03-sol-b, eval=FALSE}
 > > for( iContinent in unique(gapminder$continent) ){
-> >    tmp <- mean(subset(gapminder, continent==iContinent)$lifeExp)
-> >    cat("Average Life Expectancy in", iContinent, "is", tmp, "\n")
-> >    rm(tmp)
+> >   tmp <- gapminder[gapminder$continent == iContinent, ]   
+> >   cat(iContinent, mean(tmp$lifeExp, na.rm = TRUE), "\n")  
+> >   rm(tmp) }
```

Comment:
Closing `}` should go on its own line, per [tidyverse](https://style.tidyverse.org/syntax.html#indenting) and [Google R](https://google.github.io/styleguide/Rguide.xml#curlybraces) style guides (which we try to follow), so it looks like you have a duplicate closing `}` here.